index.css

```diff
@@ -414,3 +414,155 @@ hr {
   margin-bottom: 20px;
   padding: 0;
 }
+
+.nav-bar-desk {
+  display: none;
+}
+
+/* Media queries for desktop view */
+
+@media screen and (min-width: 768px) {
+  .nav-bar-desk {
+    display: flex;
+  }
```

Comment:
- [ ] Kindly reduce the spacing between your project cards to match the original design.

![image](https://user-images.githubusercontent.com/98453979/220967677-d49a1b6e-b326-43e7-8517-7d95ec982b60.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/220967845-cad51557-06b0-4fd7-b6ef-62ef7d481576.png)

Reply:
Hello @frank1738,

You're talking about It they should occupy 50% and 50% each?. This is how it looks on my side:

![image](https://user-images.githubusercontent.com/109441277/220987900-b61d9be8-2591-4ce0-9ea9-fa33f6eb3b59.png)


Thanks in advance!

Reply:
> Hello @frank1738,
> 
> You're talking about It they should occupy 50% and 50% each?. This is how it looks on my side:
> 
> ![image](https://user-images.githubusercontent.com/109441277/220987900-b61d9be8-2591-4ce0-9ea9-fa33f6eb3b59.png)
> 
> Thanks in advance!

Hello @J-C-S-V , I mean the vertical distance from one card to another and not the width of the cards. The distance from one card to another is quite big compared to the original design

Reply:
@frank1738 ok!

I'm going to fix that top/bottom margin for the cards then. Thank you!